src/utils/common.js

```diff
@@ -32,3 +33,26 @@ export function toQueryString(paramsObject) {
     )
     .join("&");
 }
+
+export const sanitizeHTMLContent = (htmlString) => {
```

Comment:
Can we please add types for these input parameters? We will turn on typescript strictness in the future, and we need to make this a habit.